core/app/models/spree/reimbursement.rb

```diff
@@ -136,6 +136,15 @@ def all_exchanges?
       return_items.all?(&:exchange_processed?)
     end
 
+    # Accepts all return items, saves the reimbursement, and performs the reimbursement
+    #
+    # @api public
+    def return_all!
```

Comment:
The ruby convention for methods ending in `!` is that they are a dangerous version of an equivalent method without the `!` (http://dablog.rubypal.com/2007/8/15/bang-methods-or-danger-will-rubyist)

Since there is no `return_all` method that provides this functionality, I would name this `return_all` rather than `return_all!`